OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -115,7 +118,7 @@ SimTK::ContactGeometry::TriangleMesh* ContactMesh::
     file.close();
     mesh.loadFile(filename);
     if (restoreDirectory) IO::chDir(savedCwd);
-        
+    _decorativeGeometry.reset(new SimTK::DecorativeMesh(mesh));
     return new SimTK::ContactGeometry::TriangleMesh(mesh);
```

Comment:
Yes, I debated this. Basically I didn't want the DecorativeMesh to be created and displayed unless the SimTK::ContactGeometry is created as well since otherwise the display is misleading that there's Contact while the model actually doesn't have one. That also is the reason it was removed from  testVisualization because no ContactForce was defined in the model.